SYCL/ESIMD/api/functional/ctors/ctor_default.cpp

```diff
@@ -0,0 +1,171 @@
+//==------- esimd_bit_ops.cpp  - DPC++ ESIMD on-device test ----------------==//
```

Comment:
Please, update appropriately

Reply:
I've applied this comment in this commit: 19fb0122a5e3efc3edad73f7b1ca0ad3c17c5614